proto/v1/sql_service.proto

```diff
@@ -21,6 +21,17 @@ service SQLService {
     };
   }
 
+  rpc Execute(ExecuteRequest) returns (ExecuteResponse) {
+    option (google.api.http) = {
+      post: "/v1/{name=instances/*/databases/*}:execute"
```

Comment:
Maybe we need an `additional_bindings` like `Query` for instance level execution?

Reply:
Updated